test/PracticeWebApi.Test/UnitTest1.cs

```diff
@@ -0,0 +1,10 @@
+namespace PracticeWebApi.Test;
+
+public class UnitTest1
+{
+    [Fact]
+    public void Test1()
+    {
+
```

Comment:
Could you please add a trivial assertion here? So the test is not empty and we have a way to see that the tests can fail if the assertion is false.